README.md

```diff
@@ -10,42 +10,22 @@ We are going to set up our ParallelCluster infrastructure as below:
 
-Some useful slurm commands are `sinfo` and  `squeue`. `sinfo` command displays information about slurm modes and partitions. `sinfo` command provides information about job queues currently running in the Slurm schedule. Once the job is done, slurm will generate a log file `slurm-XXXXXX.out`. You may then use `tail -f slurm-XXXXXX.out`, to inspect the job summary.
+
 
 ## Prerequisite infrastructure
-The following are required infrastructure components for ParallelCluster. You must have these component created and configured before moving on to create the cluster.
 
 ### VPC
-
-Trn1 instances of various sizes are being launched and will be available in many AWS regions. Follow the instructions [here](./examples/general/network/vpc-setup.md) to set up your VPC.
-
-### Subnets
-Once you have a VPC, you also need to create two subnets within the VPC for your HPC environment. See [AWS documentation](https://docs.aws.amazon.com/parallelcluster/latest/ug/network-configuration-v3.html#network-configuration-v3-two-subnets "Creating subnets") for creating the VPC and two subnets (**public with NAT gateway for head node, private for compute nodes**). The network configuration for this tutorial uses two subnets as described in [this diagram](https://docs.aws.amazon.com/parallelcluster/latest/ug/network-configuration-v3.html#network-configuration-v3-two-subnets "Network configuration"). As shown [here](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-subnets-commands-example.html#vpc-subnets-commands-example-launch-instance "Launch instance"), you may use AWS CLI to create a subnet within the VPC. Follow the instructions [here](./examples/general/network/subnet-setup.md) to set up your public subnet.
-
-
-### NAT gateway
-A Network Address Translation (NAT) gateway is required for compute nodes in the private subnet to connect to service outside (i.e., web access for essential software updates or packages) the VPC. This is a one-way connection such that outside services cannot connect to the nodes inside the private subnet. For convenience, choose NAT gateway option during the time you create the VPC, so that a NAT gateway is created automatically. In this example, it is created while we set up VPC as shown in [the VPC setup instructions](./examples/general/network/vpc-setup.md).
+A ParallelCluster requires a VPC. First, you must have these component created and configured before creating the cluster. [Here](./examples/general/network/vpc-setup.m) is the instruction to create a VPC. This VPC has two subnets as described in [this diagram](https://docs.aws.amazon.com/parallelcluster/latest/ug/network-configuration-v3.html#network-configuration-v3-two-subnets "Network configuration"). Second, after VPC is created, follow [this instruction](./examples/general/network/subnet-setup.md) to configure the public subnet. 
```

Comment:
Maybe replace this paragraph with:

"A ParallelCluster requires a VPC that has two subnets and a Network Address Translation (NAT) gateway as shown in the diagram above. [Here](./examples/general/network/vpc-setup.m) is the instruction to create the VPC. After the VPC is created, follow [this instruction](./examples/general/network/subnet-setup.md) to enable auto-assign public IPv4 address for the public subnet.

Reply:
Let's try to combine the two docs https://github.com/aws-neuron/aws-neuron-parallelcluster-samples/pull/10/examples/general/network/vpc-setup.md and https://github.com/aws-neuron/aws-neuron-parallelcluster-samples/pull/10/examples/general/network/subnet-setup.md

Reply:
The link "/examples/general/network/vpc-setup.m" is missing letter "d"

Reply:
Will update and combine the two md, correct link in a new commit

Reply:
fixed